scout/server/blueprints/variant/templates/variant/variant_details.html

```diff
@@ -165,24 +165,36 @@
                <td>{{ obs.homozygote|default('N/A') }}</td>
                <td>{{ obs.total|default('N/A') }}</td>
              </tr>
-           {% if obs.cases %}
+           {% if obs.cases or obs.observations == 1 %}
              <tr>
                <td colspan="4">
-                     {% for data in obs.cases %}
-                       {% if data.variant and data.variant.category == "snv"%}
-                         <a class="badge rounded-pill bg-light text-dark" target="_blank" href="{{ url_for('variant.variant', institute_id=data.case.owner, case_name=data.case.display_name, variant_id=data.variant._id) }}">{{ data.case.display_name }}</a>
-                       {% elif data.variant and data.variant.category == "sv"%}
-                         <a class="badge rounded-pill bg-light text-dark" target="_blank" href="{{ url_for('variant.sv_variant', institute_id=data.case.owner, case_name=data.case.display_name, variant_id=data.variant._id) }}">{{ data.case.display_name }}</a>
-                       {% else %}
-                         <span class="ml-3 badge rounded-pill bg-light text-dark">{{ data.case.display_name }}</span>
-                       {% endif %}
-                     {% endfor %}
-                     {% if obs.cases|length > 10 %}
-                       + more
-                     {% endif %}
-                     {% if obs.observations == 1 %}
-                       Observed only in this case
-                     {% endif %}
+                 {% if obs.observations == 1 %}
```

Comment:
I have gotten quite used to some indentation in the templates, though it is of course not functional or necessary. It does help visually grouping blocks. 😊